app/Services/Account/Settings/ExportAccount.php

```diff
@@ -188,7 +189,7 @@ private function buildInsertSQLQuery(string $tableName, string $foreignKey, arra
     private function writeToTempFile(string $sql)
     {
         Storage::disk('local')
-            ->append($this->tempFileName, $sql);
+            ->append($this->tempFileName, $sql, '');
```

Comment:
Why do you keep adding this? The `append` method only takes two arguments!

Reply:
@asbiin my comment above

Reply:
The 3rd argument is a prepend string to insert between the existing string in the file and the appended content.
See https://github.com/laravel/framework/blob/5.8/src/Illuminate/Filesystem/FilesystemAdapter.php#L305